### ISSUE TITLE
Feature request: Support camera/picture entities

### ISSUE THREAD

**Issue by chriss158** [BimmerGestalt/HassGestalt#4]
Hey Walter,

maybe something you want to add later, but it would be cool to have a look at camera entities.  maybe just a still picture, if it is not possible to show a video.

**Comment by hufman** [comment]
I agree! The API wasn't too clear on how to pull such an image, saying it's [deprecated](https://developers.home-assistant.io/docs/api/websocket#fetching-camera-thumbnails) and not offering guidance on what to use instead.
